setup.cfg

```diff
@@ -72,7 +72,8 @@ disable =
     useless-suppression,
     wildcard-import,
     not-context-manager,
-    c-extension-no-member
+    c-extension-no-member,
+    wrong-import-order
```

Comment:
i would prefer if we just configure pylint properly than disable this. which package did it complain about?